### ISSUE TITLE
Specific Multiple part skus for discounts

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#855]
_Imported from trac ticket 855.
Created by **kbower**
Opened in trac: Apr 18, 2011, 1:34:28 PM
Last modified in trac: Jan 13, 2012, 10:56:44 AM_
***
_by kbower:_
Kent or Rob,

1.       All components of the following package skus in minor code ‘BED’, ‘HBFB’ and ‘DRM’ should be considered as one piece for the purposes of discounting and the resulting discount should be applied to all components of the aforementioned package skus.

2.       All three piece components of the following package skus in minor code ‘WALL’ and ‘SECT’ should be considered as one piece for the purposes of discounting and the resulting discount should be applied to all components of the aforementioned package skus.

Thanks,

Wayne Liburd

**Comment by kbower** [comment]
_**kbower** trac comment on Jun 4, 2011, 9:49:22 AM:_

Blocked by #800

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 13, 2012, 10:56:42 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 13, 2012, 10:56:44 AM:_

Closed as duplicate of [#955](955).
